src/email-only-signup.js

```diff
@@ -1,38 +1,31 @@
 import defaultsDeep from 'lodash/object/defaultsDeep';
-import kebabCase from 'lodash/string/kebabCase';
+import helpers from './helpers'
 
-/*
- * convert a response 'code' to html message
- * @param {String} response
- * @param {String} pageLocation
- * @returns {HTMLString}
- */
-export function getResponseMsg (response, pageLocation) {
-	const responseMsg = {
-		'SUBSCRIPTION_SUCCESSFUL': 'Thanks – look out for your first briefing soon.',
-		'INVALID_REQUEST': 'Sorry, something went wrong. Please try again.',
-		'ALREADY_SUBSCRIBED': 'It looks like you’re currently receiving the daily top stories summary email. If you’re interested in getting access to more FT content, why not <a target="_blank" style="text-decoration:none;color:#27757B;" href="/products?segID=0801043&amp;segmentID=a5d15097-05ed-e096-2ac0-d693e4ece928" data-trackable="trial-existing">sign up to a 4 week trial</a>?',
-		'USER_ARCHIVED': 'It looks like you’ve signed up to the daily top stories summary email before. If you’re interested in getting access to more FT content, why not <a target="_blank" style="text-decoration:none;color:#27757B;" href="/products?segID=0801043&amp;segmentID=a5d15097-05ed-e096-2ac0-d693e4ece928" data-trackable="trial-archived">sign up to a 4 week trial</a>?',
-		'USER_NOT_ANONYMOUS': `It looks like you already have an account with us. <a target="_blank" href="/login?location=${pageLocation}" style="text-decoration:none;color:#27757B;" data-trackable="sign-in">Sign in</a>.`,
-		'USER_CONTRACT_FOUND': 'It looks like you\'re already an FT subscriber. To sign up to exclusive newsletters go to <a target="_blank" href="/newsletters" style="text-decoration:none;color:#27757B;" data-trackable="newsletters">ft.com/newsletters</a>'
-	};
-	return responseMsg[response] ? responseMsg[response] : responseMsg.INVALID_REQUEST;
-}
+let presets;
+let options;
 
-/*
- * Init component
- * @param {Element} element - the element containing the form
- * @options {Object} options - custom options
- */
-export function init (element, options={}) {
+function OEmailSignUp (element, opts) {
```

Comment:
Yeah of course, sorry I copied `o-date` so I thought it might have been an origami convention.